test/framework/src/main/java/org/elasticsearch/index/mapper/MapperTestCase.java

```diff
@@ -1243,7 +1243,7 @@ public final void testBlockLoaderFromColumnReader() throws IOException {
         testBlockLoader(true);
     }
 
-    public final void testBlockLoaderFromRowStrideReader() throws IOException {
+    public void testBlockLoaderFromRowStrideReader() throws IOException {
```

Comment:
ah! you removed `final` so you could silence it. Probably worth a comment if we're going to commit this. `final` is nice here.

Reply:
Added comment